src/components/TextStyle/TextStyle.scss

```diff
@@ -4,21 +4,21 @@ $code-padding: 0 spacing(extra-tight);
 $code-font-size: 1.15em;
 
 .variationPositive {
-  color: color('green', 'dark');
+  color: var(--p-success-text, color('green', 'dark'));
 }
 
 .variationNegative {
-  color: color('red', 'dark');
+  color: var(--p-critical-text, color('red', 'dark'));
 }
 
 .variationCode {
   position: relative;
   padding: $code-padding;
   border-radius: border-radius();
-  background-color: color('sky', 'light');
+  background-color: var(--p-surface-background, color('sky', 'light'));
   display: inline-block;
   font-size: $code-font-size;
-  box-shadow: inset 0 0 0 1px color('sky');
+  box-shadow: inset 0 0 0 1px var(--p-shadow-from-direct-light, color('sky'));
```

Comment:
I think since this is supposed to emulate a light border it should be `--p-border-subdued-on-surface`